src/test/regress/expected/distributed_functions.out

```diff
@@ -829,6 +829,248 @@ SELECT * FROM test ORDER BY id;
 (2 rows)
 
 DROP TABLE test;
+-- verify that recreating distributed functions with TABLE params gets propagated to workers
+CREATE OR REPLACE FUNCTION func_with_return_table(int)
+RETURNS  TABLE (date date)
+LANGUAGE plpgsql AS $$
+BEGIN
+    RETURN query SELECT '2011-01-01'::date;
+END;
+$$;
+SELECT create_distributed_function('func_with_return_table(int)');
+ create_distributed_function
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE OR REPLACE FUNCTION func_with_return_table(int)
+RETURNS  TABLE (date date)
+LANGUAGE plpgsql AS $$
+BEGIN
+    RETURN query SELECT '2011-01-02'::date;
+END;
+$$;
```

Comment:
we don't seem to verify that the second `CREATE OR REPLACE ` is sent to the worker.
Can we do the following to show the output changed?:
```
SELECT run_command_on_workers($$SELECT * FROM  func_with_return_table(1)$$)
```

I believe we need this in the following examples as well. Sorry that I couldn't realize in the first review. What I meant was to show that the functions on the workers are changed after the second `CREATE OR REPLACE`, it  seems querying pg_proc doesn't help.

If we want to keep the pg_proc changes, say we change the `COST` or `PROVOLATILE` etc. of the functions in the second `CREATE OR REPLACE` and get the cost/volatile from the pg_proc.

Reply:
ops, @agedemenli already added `pg_proc.prosrc` so my above comment doesn't apply